packages/integrations/sitemap/src/index.ts

```diff
@@ -88,15 +88,7 @@ const createPlugin = (options?: SitemapOptions): AstroIntegration => {
 
 					const { filter, customPages, serialize, entryLimit } = opts;
 
-					let finalSiteUrl: URL;
-					if (config.site) {
-						finalSiteUrl = new URL(config.base, config.site);
-					} else {
-						console.warn(
-							'The Sitemap integration requires the `site` astro.config option. Skipping.'
-						);
-						return;
-					}
+					let finalSiteUrl = new URL(config.base, config.site);
```

Comment:
```suggestion
					if (!config.site) {
						logger.warn('The `site` astro.config option is required. Skipping.');
						return;
					}
					let finalSiteUrl = new URL(config.base, config.site);
```

Reply:
This is already checked a couple lines above this :)